src/main/java/it/pagopa/afm/marketplacebe/service/BundleService.java

```diff
@@ -130,15 +132,16 @@ public BundleService(
      * @param pageNumber  page number
      * @return a paginated list of bundles
      */
-    public Bundles getBundles(List<BundleType> bundleTypes, String name, LocalDate validFrom, Integer limit, Integer pageNumber) {
+    public Bundles getBundles(List<BundleType> bundleTypes, String name, Instant validFrom, Integer limit, Integer pageNumber) {
+        LocalDate localValidFromDate = validFrom != null ? validFrom.atZone(ZoneId.of("Europe/Paris")).toLocalDate() : null;
```

Comment:
Why not Rome?

Reply:
In ZoneId class "Rome" is not present as a possible time zone